src/Beaches.js

```diff
@@ -64,6 +67,8 @@ class Beaches extends Component {
         <header className="header">
         <h1 className='header-title'>Beachy Keen</h1>
         </header>
+        <Filter filterByCounty={this.filterByCounty}
```

Comment:
This looks good. Good catch on including this.state.beaches so that Filter has access to it.